dosomething-dev/main.tf

```diff
@@ -78,22 +78,15 @@ module "fastly-frontend" {
 }
 
 module "fastly-backend" {
-  source = "./fastly-backend"
+  source = "../components/fastly_backends"
+  name   = "Fastly: Backends (Development)"
 
-  northstar_name    = module.northstar.name
-  northstar_domain  = module.northstar.domain
-  northstar_backend = module.northstar.backend
-
-  phoenix_name    = module.phoenix.name
-  phoenix_domain  = module.phoenix.domain
-  phoenix_backend = module.phoenix.backend
-
-  rogue_name    = module.rogue.name
-  rogue_domain  = module.rogue.domain
-  rogue_backend = module.rogue.backend
+  backends = [
+    module.northstar,
+    module.rogue,
+  ]
```

Comment:
So here is where we leverage [this work](https://github.com/DoSomething/infrastructure/pull/270/files#r490504424)?

Reply:
Yes! We send in the set of backend modules each place we use this (so e.g. `identity-dev.dosomething.org` and `activity-dev.dosomething.org`'s variables would be passed in here). We read this provided variable on the line you linked, and then iterate over that array to configure the Fastly property below.